chart/kubeapps/templates/tls-secrets.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.ingress.enabled }}
+{{- range .Values.ingress.secrets }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "kubeapps.fullname" $ }}
```

Comment:
isn't the name going to collide if there are two secrets?

Reply:
Nice catch!! I guess we can do sth like:
```
metadata:
  name: {{ .name }}
```